### PR TITLE
[DSv3] Update accuracy coverage in support matrix

### PR DESCRIPTION
**Pull request by gpolovets1**
# Description

Adding MMLU test for DeepSeek-v3 torchax backend to validate model accuracy in the support matrix.

# Tests

Please describe how you tested this change, and include any instructions and/or
commands to reproduce.

# Checklist

Before submitting this PR, please make sure:
- I have performed a self-review of my code.
- I have necessary comments in my code, particularly in hard-to-understand areas.
- I have made or will make corresponding changes to any relevant documentation.

### CODE REVIEW

**Review by gxd3 — APPROVED** [review]

File: tpu_inference/layers/vllm/quantization/fp8.py [hunk 1]
```diff
@@ -143,6 +143,14 @@ def create_weights(
         self.use_marlin = True
 
     def process_weights_after_loading(self, layer: torch.nn.Module) -> None:
+        if 'o_proj' in layer.prefix:
```
Comment (gxd3) [inline tpu_inference/layers/vllm/quantization/fp8.py]: this is for fp4 O-projection of attention layer?
Maybe have a environment variable to guard it?

**Review by jcyang43 — APPROVED** [review]

File: tpu_inference/layers/vllm/quantization/fp8.py (hunk 1 above)
Comment (gpolovets1) [reply tpu_inference/layers/vllm/quantization/fp8.py]: Sorry my local changes got tangled up - I reverted this section of code.
